client/my-sites/plugins/search-box-header/index.jsx

```diff
@@ -30,12 +29,34 @@ const SearchBox = ( { isMobile, doSearch, searchTerm, searchBoxRef, isSearching
 	);
 };
 
+const SearchLink = ( { searchTerm, currentTerm, onSelect } ) => {
+	const classes = [ 'search-box-header__recommended-searches-list-item' ];
+
+	if ( searchTerm === currentTerm ) {
+		classes.push( 'search-box-header__recommended-searches-list-item-selected' );
+
+		return <span className={ classes.join( ' ' ) }>{ searchTerm }</span>;
+	}
+
+	return (
+		<a
+			href={ `/plugins?s=${ searchTerm }` }
```

Comment:
We'll need to pass this through [localizePath](https://github.com/Automattic/wp-calypso/blob/907b78ceab529a52b7003b413a01877b8d4fb10d/client/my-sites/plugins/utils.js#L15) so we don't lose the locale parameter (ex. /es/plugins) in the URL on logged-out situation with SSR.